datasets/ONS-International-exports-of-services-from-subnational-areas-of-the-UK/main.py

```diff
@@ -17,106 +17,279 @@
 
 # +
 from gssutils import *
+import json
 
-scraper = Scraper('https://www.ons.gov.uk/businessindustryandtrade/internationaltrade/datasets/internationalexportsofservicesfromsubnationalareasoftheuk')
-tabs = { tab.name: tab for tab in scraper.distributions[0].as_databaker() }
+cubes = Cubes("info.json")
+trace = TransformTrace()
+info = json.load(open('info.json'))
+scraper = Scraper(seed = 'info.json')
+scraper
 # -
 
+tabs = { tab.name: tab for tab in scraper.distributions[0].as_databaker() }
```

Comment:
this scraper.distributions[0] pattern is more legacy code we need to stop using. could you use the following please:

```
`distribution = scraper.distribution(latest=True)
tabs = { tab.name: tab for tab in distribution.as_databaker() }`
```

Reply:
Legacy code is replaced